### PR TITLE
Add createTreeWithRoot instruction parsing

### PR DESCRIPTION
**Pull request by n00m4d**

### CODE REVIEW

File: blockbuster/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "blockbuster"
 description = "Metaplex canonical program parsers, for indexing, analytics etc...."
-version = "2.3.0"
+version = "2.4.0"
```
Comment (rwwwx) [inline blockbuster/Cargo.toml:4]: I raised the crate version, but I am not sure that it's necessary. So if it's redundant plz inform me

File: blockbuster/Cargo.lock
Comment (StanChe) [inline blockbuster/Cargo.lock:1]: looks like this one is safe to drop

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,4 +13,5 @@ jobs:
       - uses: actions/checkout@v3
       - name: Test
         run: |
+          cd ./blockbuster
```
Comment (StanChe) [inline .github/workflows/test.yml:16]: this change may be dropped

File: blockbuster/Cargo.toml [hunk 3]
```diff
@@ -11,10 +11,14 @@ readme = "../README.md"
...
-mpl-bubblegum = "1.2.0"
-mpl-core = { version = "0.5.0", features = ["serde"] }
+
+# Update these deps to original SPL crate after merging 'feature/init_with_root' changes into original repo.
+spl-account-compression = { git = "https://github.com/StanChe/solana-program-library.git", branch = "feature/init_with_root", features = ["no-entrypoint"] }
+spl-noop = { git = "https://github.com/StanChe/solana-program-library.git", branch = "feature/init_with_root", features = ["no-entrypoint"] }
+mpl-bubblegum = { git = "https://github.com/adm-metaex/mpl-bubblegum", branch = "feature/cleanup", features = ["test-sbf", "serde"] }
+# ^^^
+
+mpl-core = { version = "=0.8.0-beta.1", features = ["serde"] }
```
Comment (StanChe) [inline blockbuster/Cargo.toml:21]: Why beta? Can't we use a stable version?

Comment (RequescoS) [reply blockbuster/Cargo.toml:21]: Reference implementation use the same version for  
mpl-core

**Pull request merged**